lib/origen/version_string.rb

```diff
@@ -75,7 +75,9 @@ def valid?(_options = {})
     # Returns true if the version is a semantic format version number
     def semantic?
       !!(self =~ /^v?\d+\.\d+\.\d+$/ ||
-         self =~ /^v?\d+\.\d+\.\d+\.(dev|pre)\d+$/
+         self =~ /^v?\d+\.\d+\.\d+\.(dev|pre)\d+$/ ||
+         self =~ /^\d+\.\d+\.\d+$/ ||
```

Comment:
The v is already optional above - v?, instead of just v.
I think all the change in this file are un-necessary.

Reply:
OK